Link-Preview/linkPreview.py

```diff
@@ -0,0 +1,145 @@
+import requests
+import json
+import os
+import time
+from bs4 import BeautifulSoup
+
+# to scrape title
+
+
+def getTitle(soup):
+    ogTitle = soup.find("meta", property="og:title")
+
+    twitterTitle = soup.find("meta", attrs={"name": "twitter:title"})
+
+    documentTitle = soup.find("title")
+    h1Title = soup.find("h1")
+    h2Title = soup.find("h2")
+    pTitle = soup.find("p")
+
+    res = ogTitle or twitterTitle or documentTitle or h1Title or h2Title or pTitle
+    res = res.get_text() or res.get("content", None)
+
+    if (len(res) > 60):
+        res = res[0:60]
+    if (res == None or len(res.split()) == 0):
+        res = "Not available"
+    return res.strip()
+
+# to scrape page description
+
+
+def getDesc(soup):
+    ogDesc = soup.find("meta", property="og:description")
+
+    twitterDesc = soup.find("meta", attrs={"name": "twitter:description"})
+
+    metaDesc = soup.find("meta", attrs={"name": "description"})
+
+    pDesc = soup.find("p")
+
+    res = ogDesc or twitterDesc or metaDesc or pDesc
+    res = res.get_text() or res.get("content", None)
+    if (len(res) > 60):
+        res = res[0:60]
+    if (res == None or len(res.split()) == 0):
+        res = "Not available"
+    return res.strip()
+
+# to scrape image link
+
+
+def getImage(soup, url):
+    ogImg = soup.find("meta", property="og:image")
+
+    twitterImg = soup.find("meta", attrs={"name": "twitter:image"})
+
+    metaImg = soup.find("link", attrs={"rel": "img_src"})
+
+    img = soup.find("img")
+
+    res = ogImg or twitterImg or metaImg or img
+    res = res.get("content", None) or res.get_text() or res.get("src", None)
+
+    count = 0
+    for i in range(0, len(res)):
+        if (res[i] == "." or res[i] == "/"):
+            count += 1
+        else:
+            break
+    res = res[count::]
+    if ((not res == None) and ((not "https://" in res) or (not "https://" in res))):
+        res = url + "/" + res
+    if (res == None or len(res.split()) == 0):
+        res = "Not available"
+
+    return res
+
+# print dictionary
+
+
+def printData(data):
+    print("\nTitle : ", data["title"])
```

Comment:
```suggestion
    for item in data.items():
        print(f'{item[0]}: {item[1]}')
```
Instead of manually printing the dict, you can do something like this.